### PR TITLE
v5.17.4

### PR DESCRIPTION
**Pull request by flaviendelangle**
See #6254 for the v6 alpha release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,34 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.4
+
+_Sep 22, 2022_
+
+We'd like to offer a big thanks to the 4 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Persian (fa-IR) locale to the pickers (#6181) @alexfauquette
```
Comment (alexfauquette) [inline CHANGELOG.md:12]:
```suggestion
- 🌍 Add Persian (fa-IR) locale to the pickers (#6181) @fakhamatia
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,34 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.4
+
+_Sep 22, 2022_
+
+We'd like to offer a big thanks to the 4 contributors who made this release possible. Here are some highlights ✨:
```
Comment (alexfauquette) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,34 @@
...
+#### Changes
+
+- [DataGrid] Improve Polish (pl-PL) locale on the data grid (#6245) @grzegorz-bach
+
+### `@mui/x-date-pickers@v5.0.3` / `@mui/x-date-pickers-pro@v5.0.3`
+
+#### Changes
+
+- [pickers] Add Finnish (fi-FI) locale to pickers (#6219) (#6230) @PetroSilenius
+- [pickers] Add Persian (fa-IR) locale to the pickers (#6181) @alexfauquette
```
Comment (alexfauquette) [inline CHANGELOG.md:26]:
```suggestion
- [pickers] Add Persian (fa-IR) locale to the pickers (#6181) @fakhamatia
```

**Review by alexfauquette — APPROVED** [review]

**Comment by cherniavskii** [comment]
I've merged master to include https://github.com/mui/mui-x/pull/6251 and https://github.com/mui/mui-x/pull/6239

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,39 @@
...
+- [pickers] Fix usage with Typescript 4.8 (#6229) @flaviendelangle
+- [YearPicker] Scroll to the current year even with `autoFocus=false` (#6224) @alexfauquette
+
+### Docs
+
+- [docs] Fix 301 link @oliviertassinari
```
Comment (m4theushw) [inline CHANGELOG.md:33]:
```suggestion
- [docs] Fix 301 link (#6239) @oliviertassinari
```

**Review by m4theushw — APPROVED** [review]

**Review by MBilalShafi — APPROVED** [review]

**Pull request merged**